packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -664,6 +664,187 @@ describe('RichTextEditor', () => {
                 'P'
             ]);
         });
+
+        describe('Absolute link interactions in the editor', () => {
+            describe('various absolute links without other nodes and marks', () => {
+                const supportedAbsoluteLink: { name: string }[] = [
+                    { name: 'https://nimble.ni.dev/ ' },
+                    { name: 'HTTPS://NIMBLE.NI.DEV ' },
+                    { name: 'HttPS://NIMBLE.ni.DEV ' },
+                    { name: 'http://nimble.ni.dev/ ' },
+                    { name: 'HTTP://NIMBLE.NI.DEV ' },
+                    { name: 'HttP://nimble.NI.dev ' },
+                    {
+                        name: 'https://www.example.com/path/=equals&ampersand?question$dollar+plus,comma;semicolon@At '
+                    },
+                    { name: 'https://example.com/my%20page.html ' },
+                    { name: 'https://example.com/smiley😀.html ' },
+                    { name: 'https://example.com/пример.html ' }
+                ];
+
+                const focused: string[] = [];
+                const disabled: string[] = [];
+                for (const value of supportedAbsoluteLink) {
+                    const specType = getSpecTypeByNamedList(
+                        value,
+                        focused,
+                        disabled
+                    );
+                    specType(
+                        `should change the ${value.name} to "a" tag when it is a valid absolute link`,
+                        // eslint-disable-next-line @typescript-eslint/no-loop-func
+                        async () => {
+                            await pageObject.setEditorTextContent(value.name);
+
+                            expect(pageObject.getEditorTagNames()).toEqual([
+                                'P',
+                                'A'
+                            ]);
+                            expect(pageObject.getEditorLeafContents()).toEqual([
+                                // Name without the trailing space used by the editor to trigger conversion to a link
+                                value.name.slice(0, -1)
+                            ]);
+                        }
+                    );
+                }
+            });
+
+            it('should have the right attributes to the "a" tag', async () => {
```

Comment:
nit: prefer specific descriptions rather than words like "right" or "correct" (we haven't been too strict about this but I happened to notice this one)

```suggestion
            it('the "a" tag should have href and rel attributes', async () => {
```